src/components/LeftSidebar/SearchConversationsResults/SearchConversationsResults.vue

```diff
@@ -0,0 +1,332 @@
+				? t('spreed', 'Teams')
+				: t('spreed', 'Other sources')
+		}
+	}
+})
+
+const computedStyleFooter = computed(() => {
+	return {
+		marginBlockStart: isCompact.value ? '0' : '18px', // 54px (item height) - 36px (current height)
+	}
+})
+
+const iconSize = computed(() => isCompact.value ? AVATAR.SIZE.COMPACT : AVATAR.SIZE.DEFAULT)
+</script>
+
+<template>
+	<RecycleScroller ref="scroller"
+		item-tag="ul"
+		:items="searchResultsVirtual"
+		type-field="type"
+		:item-size="itemSize">
+		<template #default="{ item }">
+			<Conversation v-if="item.type === 'conversation'"
+				:key="`conversation_${item.id}`"
+				:ref="`conversation-${item.object.token}`"
+				:item="item.object"
+				:compact="isCompact"
+				@click="emit('abort-search')" />
+			<NcListItem v-else-if="item.type === 'listItem'"
+				:name="searchText"
+				:compact="isCompact"
+				data-nav-id="conversation_create_new"
+				@click="emit('create-new-conversation', searchText)">
+				<template #icon>
+					<IconChatPlus :size="iconSize" />
+				</template>
+				<template v-if="!isCompact" #subname>
+					{{ t('spreed', 'New group conversation') }}
+				</template>
+			</NcListItem>
+			<Conversation v-else-if="item.type === 'open_conversation'"
+				:key="`open-conversation_${item.id}`"
+				:item="item.object"
+				is-search-result
+				:compact="isCompact"
+				@click="emit('abort-search')" />
+			<NcAppNavigationCaption v-else-if="item.type === 'caption'" 
+				:name="item.name" 
+				tabindex="-1"
+				:style="{
+					height: itemSize + 'px',
+					alignItems: isCompact ? 'unset' : 'self-end',
+				}" />
+			<Hint v-else-if="item.type === 'hint'" :hint="item.hint" />
+			<NcListItem v-else-if="item.type === 'user'"
+				:key="`user_${item.id}`"
+				:data-nav-id="`user_${item.id}`"
+				:name="item.object.label"
+				:compact="isCompact"
+				@click="emit('create-and-join-conversation', item.object)">
+				<template #icon>
+					<AvatarWrapper v-bind="iconData(item.object)" />
+				</template>
+				<template v-if="!isCompact" #subname>
+					{{ t('spreed', 'New private conversation') }}
+				</template>
+			</NcListItem>
+			<NcListItem v-else-if="item.type === 'group'"
+				:key="`group_${item.id}`"
+				:data-nav-id="`group_${item.id}`"
+				:name="item.object.label"
+				:compact="isCompact"
+				@click="emit('create-and-join-conversation', item.object)">
+				<template #icon>
+					<ConversationIcon :item="iconData(item.object)" :size="iconSize" />
+				</template>
+				<template v-if="!isCompact" #subname>
+					{{ t('spreed', 'New group conversation') }}
+				</template>
+			</NcListItem>
+			<NcListItem v-else-if="item.type === 'circle'"
+				:key="`circle_${item.id}`"
+				:data-nav-id="`circle_${item.id}`"
+				:name="item.object.label"
+				:compact="isCompact"
+				@click="emit('create-and-join-conversation', item.object)">
+				<template #icon>
+					<ConversationIcon :item="iconData(item.object)" :size="iconSize" />
+				</template>
```

Comment:
Instead of calling a method in a template (and having a new object by new reference each re-render), let's add this data into `item` when we prepare the item